### DISCUSSION TITLE
Profiles/presets disapear

### DISCUSSION THREAD

**Discussion by gelimelech** [OrcaSlicer/OrcaSlicer#5022] [General]
I had multiple presets for each nozzle size and it disappeared.
ive imported the configs again but the presets didnt appear.
Any idea?
![image](https://github.com/SoftFever/OrcaSlicer/assets/42086245/3c0da0b4-8e37-402f-9e39-ef24ffd235c5)
![image](https://github.com/SoftFever/OrcaSlicer/assets/42086245/4086def3-acd3-4d43-823e-4313ed474ac9)

**Comment by DarkSyd007** [comment]
Same here.  The system presets for the .4 variant of my printer are gone now too.  I re-added my Ender 3 V3 SE .4 nozzle, and all I get are "default presets".  If I add the "new" .2, .6, or .8 printer profile, I get the expected included system presets.

I'm using the portable windows version.

**Comment by DarkSyd007** [comment]
My problem was same as here:  https://github.com/SoftFever/OrcaSlicer/discussions/4828

**Reply by gelimelech** [reply]
thanks!
i dont know what i did, but now my presets are back :)

**Reply by Billones142** [reply]
mine too, looks like if you open orca they dont appear, but once you restart it they appear again, really annoying
